### ISSUE TITLE
VBA runs successfully only once in FireFox 16.0.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#11]
```
Operating system : Windows XP SP3
.Net Framework version : 3.5 SP1
Office Version : 2010
SeleniumWrapper version : 1.0.6.2

FireFox version : 16.0.1

What steps will reproduce the problem?
1. In Excel VBE, run the Example code.
2. Wait until it stops
3. Run the code again

Firefox Window pops up again but nothing happens, and later Excel says timeout.
```

Original issue reported on code.google.com by `zope...@gmail.com` on 21 Oct 2012 at 6:21

**Comment by GoogleCodeExporter** [comment]
Original comment by `florentbr` on 21 Oct 2012 at 5:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 1.0.6.3
```

Original comment by `florentbr` on 22 Oct 2012 at 12:30
- Changed state: **Fixed**
